blogposts/governance-app.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Building a Trustless Decentralised Governance Application 🌐'
+date: '2024-07-09T10:41:00'
+tags: ['Blockchain', 'Node.js', 'Decentralized', 'Aeternity', 'Vue.js', 'web3']
+author: 'Keno Dreßel'
+previewText: 'Explore the development, features, and impact of the aeternity Governance App.'
+previewImage: '/blogposts/preview/governance-app.png'
+---
+
+<p className="screenshots">
+  <img src="/blogposts/governance-app/img.png" />
+  <img src="/blogposts/governance-app/img_1.png" /> 
+  <img src="/blogposts/governance-app/img_2.png" />
+</p>
+
+**TLDR**: We made a decentralized voting app which you can try it out at [governance.aeternity.com](https://governance.aeternity.com)!
+
+## 🏛️ Introduction and Background
+
+aeternity, a platform committed to creating a more equitable online ecosystem through scalable, open-source, and
```

Comment:
```suggestion
Aeternity, a platform committed to creating a more equitable online ecosystem through scalable, open-source, and
```

Reply:
aeternity is a brand name, so it feels a bit weird but they consistenly write their name in lowercase